### PR TITLE
Update action and add merge conflict labeler

### PR DESCRIPTION
**Pull request by Gauvino**
#### Description
Added a merge conflict labeler workflow for all the PR when a merge conflict is detected so people can resolve it when commit on develop branch is made

`merge-conflict` label need to be made before merging this PR

Update action version to remove the warning and adapt parameters on labeler

### CODE REVIEW

File: .github/workflows/conflict_labeler.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  pull_request_target:
+  issue_comment:
+
+permissions: {}
+
+jobs:
+  label:
+    name: Labeling
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Sonarr/Sonarr' }}
```
Comment (markus101) [inline .github/workflows/conflict_labeler.yml:16]: This will prevent it from being used down stream without changes, which makes it a little awkward. Is there a way to check if it's a fork and not run on forks?

File: .github/workflows/conflict_labeler.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - name: Apply label
+        uses: eps1lon/actions-label-merge-conflict@v3
+        if: ${{ github.event_name == 'push' || github.event_name == 'pull_request_target'}}
+        with:
+          dirtyLabel: 'merge conflict'
```
Comment (markus101) [inline .github/workflows/conflict_labeler.yml:22]: This would be more in line with our existing labels.

```suggestion
          dirtyLabel: 'merge-conflict'
```

File: .github/workflows/conflict_labeler.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          commentOnDirty: 'This pull request has merge conflicts. Please resolve the conflicts so the PR can be successfully reviewed and merged.'
```
Comment (markus101) [inline .github/workflows/conflict_labeler.yml:23]: Is it possible to exclude the comment? I don't think it is necessary as authors will get notifications when the label is added.

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -236,7 +236,7 @@ jobs:
 
     steps:
       - name: Notify
-        uses: tsickert/discord-webhook@v5.3.0
+        uses: tsickert/discord-webhook@v6.0.0
```
Comment (markus101) [inline .github/workflows/build.yml:239]: Please do these separate version bumps in a different PR.

File: .github/workflows/conflict_labeler.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
+name: Merge Conflict Labeler
+
+on:
+  push:
+    branches:
+      - develop
+  pull_request_target:
+  issue_comment:
```
Comment (markus101) [inline .github/workflows/conflict_labeler.yml:8]: Is this necessary?

**Review by markus101 — CHANGES_REQUESTED** [review]
I agree with what Stevie and Bodgan mentioned on Discord, but I do like the automation of labeling and being able to filter out PRs that need reviews.

@stevietv @mynameisbogdan what do you think about having it label and exclude the comment?

File: .github/workflows/build.yml (hunk 4 above)
Comment (Gauvino) [reply .github/workflows/build.yml:239]: will do it no problem

File: .github/workflows/conflict_labeler.yml (hunk 3 above)
Comment (Gauvino) [reply .github/workflows/conflict_labeler.yml:23]: I don't think people get the notification when their PR is labeled, the only notification you get on a PR is comment and commit

File: .github/workflows/conflict_labeler.yml (hunk 1 above)
Comment (Gauvino) [reply .github/workflows/conflict_labeler.yml:16]: I don't think so this is the best I found to do it, let me know if you know other way to do it

File: .github/workflows/conflict_labeler.yml (hunk 5 above)
Comment (Gauvino) [reply .github/workflows/conflict_labeler.yml:8]: the name is issue_comment but it work also for PR see screenshot 

chrome_IgQXjJIRmF

File: .github/workflows/conflict_labeler.yml (hunk 3 above)
Comment (Gauvino) [reply .github/workflows/conflict_labeler.yml:23]: So you would prefer no comment at all ?

File: .github/workflows/conflict_labeler.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    name: Labeling
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'Sonarr/Sonarr' }}
+    steps:
+      - name: Apply label
+        uses: eps1lon/actions-label-merge-conflict@v3
+        if: ${{ github.event_name == 'push' || github.event_name == 'pull_request_target'}}
+        with:
+          dirtyLabel: 'merge-conflict'
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
```
Comment (markus101) [inline .github/workflows/conflict_labeler.yml:23]: To match other files in the repo:

```suggestion
          repoToken: "${{ secrets.GITHUB_TOKEN }}"
          
```

**Review by markus101 — APPROVED** [review]
Thanks for the cleanup.
